desktop-app/src/renderer/components/Previewer/Device/index.tsx

```diff
@@ -495,6 +495,61 @@ const Device = ({ isPrimary, device, setIndividualDevice }: Props) => {
   const scaledHeight = height * zoomfactor;
   const scaledWidth = width * zoomfactor;
 
+  useEffect(() => {
+    const webview = ref.current;
```

Comment:
I would say, we don't need this button inside the view. We don't want to pollute the user space with the app code. 

The icon in the toolbar is good enough and this just feels like a redundant cta. Can you please remove this?

Reply:
Took a stab at it myself.